apps/meteor/app/theme/client/imports/general/base_old.css

```diff
@@ -1815,7 +1815,7 @@
 	}
 }
 
-@media (width <= 780px) {
+@media (max-width: 767px) {
```

Comment:
I don't understand, why are these changes relevant?

Reply:
Not fully related to the PR title, they're a small fix for other regression.
Filipe can explain better, but AFAIK, Safari don't support the `w < x` instruction so some breakpoints were not being applied.

Reply:
You are right Kevin. For this specific one, I also change the breakpoint value to match the fuselage value and fix some position issues.